web-animations-1/Overview.bs

```diff
@@ -459,39 +459,41 @@ Specification conventions {#spec-conventions}
 Web Animations model overview {#web-animations-model-overview}
 =============================
 
-<div class=informative-bg>
-<em>This section is non-normative</em>
-
   At a glance, the [=Web Animations model=] consists
   of two largely independent pieces:
-  a <em>timing model</em> and an <em>animation model</em>.
+  a [=timing model=] and an [=animation model=].
   The roles of these pieces are as follows:
 
-  : Timing model
+  : <dfn id="timing-model-dfn">timing model</dfn>
   ::
       Takes a moment in time and converts it to a proportional distance
       within a single iteration of an animation,
-      called the <em>iteration progress</em>.
-      The <em>iteration index</em> is also recorded
+      called the <dfn>iteration progress</dfn>.
```

Comment:
The definition for iteration progress should remain in 4.11, not here.

Reply:
Ok, so reviewing the change... I disagree. :) I don't mind adding a cross-reference, but I want the DFN here for two reasons:

1. It's the actual defining instance of the term. 4.11 says how to calculate it (sorta), but doesn't say what it *is*. 

2. For exactly that reason, 4.11 is confusing by itself. I land there and I don't understand what I'm looking at. I keep rewinding up the call stack, but I'm deep in the middle of a bunch of calculations for which I have no context. If it's a link back to the core definition, at least I can click on it to understand what's happening.

Looking through the spec, moving the DFN to 4.9 might make sense; it would require some tweaking of the prose to introduce an actual definition, but it seems to have appropriate context for one. Wdyt?

Reply:
Ok, maybe I'm overreacting after encountering so many cases where CSS spec definitions just throw a term out there without actually defining it.

My understanding was this section was defining the timing model. The appearance of _iteration progress_ was just to aid that description. The actual definition of the iteration should come later.

I'm not sure about 4.9 (the current draft has 4.9 as "4.9 Direction control" which I'm pretty sure it not what you meant). I could see "4.5 Animation effects" as a possibility, since the iteration progress is a property of an animation effect.

Alternatively, we could just beef up the description of the iteration progress in 4.11. For example, for the _overall progress_ we have:

> The **overall progress** describes the number of iterations that have completed (including partial iterations) and is defined as follows:

Reply:
> Ok, maybe I'm overreacting after encountering so many cases where CSS spec definitions just throw a term out there without actually defining it.

That seems like an editorial error that we should fix. :)

> I'm not sure about 4.9 (the current draft has 4.9 as "4.9 Direction control" which I'm pretty sure it not what you meant).

I was referencing https://www.w3.org/TR/web-animations-1/#core-animation-effect-calculations so actually, I guess it's 4.8 :)

So I think what I'll do is leave the DFN where it is for now, and then maybe switch it up once I get something to anchor it in maybe around 4.8. (since I'm going top down, essentially)

Reply:
Ok, fair enough. If you wanted to drop in a note so we don't forget to move the definition, that would be nice. I'd like to make sure we define the term properly in a suitable place, instead of it just being a side-effect of another definition.